src/covid19model/models/base.py

```diff
@@ -317,16 +317,29 @@ def date_to_diff(self, start_date, date, excess_time):
         """
         return int((pd.to_datetime(date)-pd.to_datetime(start_date))/pd.to_timedelta('1D'))+excess_time
 
-    def sim(self, time, excess_time=None, start_date='2020-03-15'):
+    def sim(self, time, excess_time=None, start_date='2020-03-15', N=1, draw_fcn=None, samples=None):
         """
-        Run a model simulation for the given time period.
+        Run a model simulation for the given time period. Can optionally perform N repeated simulations of time days.
+        Can use samples drawn using MCMC to perform the repeated simulations.
+
 
         Parameters
         ----------
         time : int or list of int [start, stop]
             The start and stop time for the simulation run.
             If an int is specified, it is interpreted as [0, time].
 
+        N : int
+            Number of repeated simulations. One by default.
+
+        draw_fcn : function
+            A function which takes as its input the dictionary of model parameters and the dictionary of sampled parameter values and assings these samples to the model parameter dictionary ad random.
```

Comment:
assigns

Reply:
I suppose this is an integration of `draw_sample_COVID19_SEIRD` that is now in `utils`?